.github/workflows/ci.yml

```diff
@@ -361,4 +361,4 @@ jobs:
         uses: taiki-e/install-action@cargo-hack
 
       - name: Check feature powerset
-        run: cargo hack check --feature-powerset --no-dev-deps
+        run: cargo hack check --locked --feature-powerset --no-dev-deps
```

Comment:
IIRC I had to explicitly leave out `--locked` for the equiv. in Rustls land:
https://github.com/rustls/rustls/blob/124f31123cfcb7c03ba29d17a1755d247b073490/.github/workflows/daily-tests.yml#L138

If I remember right it ended up stomping over its own lockfile working through the powerset.

Reply:
https://github.com/rustls/rustls/commit/1a939124e8b8a72f21bdb557b8d80dc6eef72522

Reply:
Aha! Dropped this --locked too.